### DISCUSSION TITLE
Lesson 3 - cannot find "Get the latest price" on Chainlink

### DISCUSSION THREAD

**Discussion by sherryyw** [smartcontractkit/full-blockchain-solidity-course-py#1801] [Q&A]
I'm on Lesson 3. It seems Chainlink has edited its UI? Where is the remix button I can click to transport the codes to my Remix? Anyone knows?

**Comment by cromewar** [comment]
Hello @sherryyw 
Could you be more specific with your question? if you provide some screenshots would be great.
